### ISSUE TITLE
[CI] DockerTests test022InstallPluginsFromLocalArchive failing

### ISSUE THREAD

**Issue by davidkyle** [elastic/elasticsearch#80147]
**Build scan:**
https://gradle-enterprise.elastic.co/s/tkt2mld33xlj4/tests/:qa:os:destructiveDistroTest.default-dockerCloudEss/org.elasticsearch.packaging.test.DockerTests/test022InstallPluginsFromLocalArchive

**Reproduction line:**
`null`

**Applicable branches:**
master

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.packaging.test.DockerTests&tests.test=test022InstallPluginsFromLocalArchive

**Failure excerpt:**
```
java.lang.IllegalStateException: failed to load plugin analysis-icu while checking for jar hell
Likely root cause: java.nio.file.NoSuchFileException: /usr/share/elasticsearch/plugins/.installing-10993802637684828079/lucene-analysis-icu-9.0.0-snapshot-2719cf6630e.jar
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:149)
	at java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
	at java.base/java.nio.file.Files.readAttributes(Files.java:1851)
	at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1264)
	at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:709)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:243)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:172)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:347)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:318)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:257)
	at org.elasticsearch.jdk.JarHell.checkJarHell(JarHell.java:180)
	at org.elasticsearch.plugins.PluginsService.checkBundleJarHell(PluginsService.java:645)
	at org.elasticsearch.plugins.PluginsService.loadBundles(PluginsService.java:487)
	at org.elasticsearch.plugins.PluginsService.<init>(PluginsService.java:165)
	at org.elasticsearch.node.Node.<init>(Node.java:378)
	at org.elasticsearch.node.Node.<init>(Node.java:278)
	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:223)
	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:223)
	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:333)
	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:164)
	at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:155)
	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:77)
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:112)
	at org.elasticsearch.cli.Command.main(Command.java:77)
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:120)
	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:78)
For complete error details, refer to the log at /usr/share/elasticsearch/logs/docker-cluster.log]
[2021-11-01T12:13:45,861][ERROR][o.e.p.u.d.DockerShell    ] [TEST-DockerTests.test022InstallPluginsFromLocalArchive-seed#[6FD6C17025070918]-workerest_thread_info] Command [/usr/share/elasticsearch/bin/elasticsearch-plugin list] failed.

```

**Comment by droberts195** [comment]
This failed again today in https://gradle-enterprise.elastic.co/s/mv75aloshjkm2/tests/:qa:os:destructiveDistroTest.default-dockerCloudEss/org.elasticsearch.packaging.test.DockerTests/test022InstallPluginsFromLocalArchive

**Comment by pugnascotia** [comment]
It's possible this will have been fixed by #77544, but I can't be sure right now. If this stops failing for long enough, then maybe we can close the issue.

**Comment by mark-vieira** [comment]
> It's possible this will have been fixed by #77544, but I can't be sure right now. If this stops failing for long enough, then maybe we can close the issue.

I think so given it [hasn't failed since](https://gradle-enterprise.elastic.co/scans/tests?search.relativeStartTime=P28D&search.timeZoneId=America/Los_Angeles&tests.container=org.elasticsearch.packaging.test.DockerTests&tests.sortField=FAILED&tests.test=test023InstallPluginsFromLocalArchive&tests.unstableOnly=true).
